playbooks/common/openshift-cluster/upgrades/v3_9/upgrade_control_plane.yml

```diff
@@ -72,26 +58,12 @@
 - name: Configure the upgrade target for the common upgrade tasks 3.9
   hosts: oo_masters_to_config:oo_etcd_to_config:oo_lb_to_config
   tasks:
-  - meta: clear_facts
   - set_fact:
       openshift_upgrade_target: '3.9'
       openshift_upgrade_min: '3.8'
       openshift_release: '3.9'
```

Comment:
Need to use user's requested version of openshift release.

Reply:
I don't care for setting a bunch of extra booleans where it's not clear what they are for or where they are used.  I can update this to preserve `openshift_release` but we will have another situation where an inventory variable could be undefined and with the double upgrade we can't preserve an undefined variable.  We could turn this process around and only pass new vars into the openshift_version role and not directly operate on the inventory vars, but would require variable changes throughout.